.gitignore

```diff
@@ -8,4 +8,7 @@ storybook-static
 
 # Editor
 .vscode
-.DS_Store
```

Comment:
I think I'd like to encourage you to leave new lines at the end of your files, so that Git and other unix powered tools would not consider this line to have changed

Reply:
Would love to - if we're able to add linting to this project, that'd be great, but I'm not sure how we'd do so in a way consistent w/ internal projects, without perhaps moving our eslint package to GH...

Reply:
Fixing for now manually...

Reply:
@hungsu - fixed for all files I could find - 7cea1d59bb630c11e7b144223048c882b741e793